scripts/rddepman.ts

```diff
@@ -0,0 +1,145 @@
+// A cross-platform script to check if newer versions of
+// external dependencies are available.
+
+import { spawnSync } from 'child_process';
+import path from 'path';
+
+import { LimaAndQemu, AlpineLimaISO } from 'scripts/dependencies/lima';
+import { MobyOpenAPISpec } from 'scripts/dependencies/moby-openapi';
+import * as tools from 'scripts/dependencies/tools';
+import { WSLDistro, HostResolverHost, HostResolverPeer } from 'scripts/dependencies/wsl';
+import {
+  DependencyVersions, readDependencyVersions, writeDependencyVersions, Dependency, AlpineLimaISOVersion, getOctokit,
+} from 'scripts/lib/dependencies';
+
+const MAIN_BRANCH = 'main';
+const GITHUB_OWNER = 'rancher-sandbox';
+const GITHUB_REPO = 'rancher-desktop;';
```

Comment:
It would be nice to be able to override via environment (for testing); something like
```ts
const GITHUB_OWNER = process.env.GITHUB_OWNER || 'rancher-sandbox';
```
(This would mainly be for testing — I'd be able to run this against my fork and not worry about making a pile of PRs)

Reply:
That's a good idea. Done.